### PR TITLE
Merge pull request #187 from horacioh/patch-1

### PR DESCRIPTION
**Pull request by horacioh**
adding this config to the project will avoid problems when adding MDX files to storybook

solves #165

### CODE REVIEW

File: content/design-systems-for-developers/react/en/document.md [hunk 1]
```diff
@@ -156,6 +156,21 @@ First, let’s take control of the Avatar doc generation from the default. Regis
...
+Let's add some configuration options to the addons-docs preset.
+
+```javascript
+module.exports = [
+  {
+    name: '@storybook/addon-docs/react/preset',
+    options: {
+      configureJSX: true,
+      babelOptions: {},
+      sourceLoaderOptions: null,
```
Comment (kylesuss) [inline content/design-systems-for-developers/react/en/document.md]: Thanks for the PR! Couple things:

1. Just `configureJSX` is required here for MDX support, right?
2. I see another part of this file that seems to duplicate the preset. Find it here:

https://github.com/chromaui/learnstorybook.com/blob/2863cbecd87ce23c9d7c86b3b2e5750435284091/content/design-systems-for-developers/react/en/document.md#write-stories-generate-docs

Feels like the area linked above should instead be updated to include the `options` object with only the option that we need for MDX support. What do you think?

Comment (horacioh) [reply content/design-systems-for-developers/react/en/document.md]: that sounds great!
let me do the fix

**Comment by horacioh** [comment]
@kylesuss done! let me know what you think :)

**Comment by kylesuss** [comment]
@horacioh love it, thanks. LGTM. @tmeasday care to sanity check?

**Comment by tmeasday** [comment]
I think @shilman was still determining the best way to go here but I think this is probably the right thing to do in the short term?

**Comment by domyen** [comment]
@tmeasday looks like this is ready to go. Do we need to update the code example?

**Comment by horacioh** [comment]
@domyen maybe it does... https://github.com/chromaui/learnstorybook-design-system/blob/master/.storybook/presets.js

I can do a PR on it too if needed

**Comment by tmeasday** [comment]
Thanks @horacioh -- I guess we need to rebase the code so the series of commit stay "correct". That requires some git "surgery" and is a bit of a pain, to be honest. I'll try and get to it soon.

**Comment by tmeasday** [comment]
Apologies all, I should have merged this. @shilman can you check if this syntax is correct?

**Comment by shilman** [comment]
LGTM. merging!

**Pull request merged**